examples/alanine_dipeptide_tps/AD_tps_4_advanced.ipynb

```diff
@@ -1,5 +1,18 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Advanced analysis techniques\n",
+    "\n",
+    "Now we'll move on to a few more advanced analysis techniques. (These are discussed in Paper II.)\n",
+    "\n",
+    "With the fixed path length ensemble, we should check for recrossings. To do this, we create an ensemble which represents the recrossing paths: a frame in $\\beta$, possible frames in neither $\\alpha$ nor $\\beta$, and then a frame in $\\alpha$.\n",
+    "\n",
+    "Then we check whether any subtrajectory of a trial trajectory matches that ensemble, by using the `Ensemble.split()` function. We can then further refine to see which steps that included trials with recrossings were actually accepted."
+   ]
+  },
```

Comment:
This notebook should be updated to use the new storage filenames, `print` should be functions